.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: false
```

Comment:
@henrahmagix I was talking to Dr G and he says we do want this and it make travis run faster :)